scripts/test-vector-validation/main.js

```diff
@@ -18,6 +18,18 @@ const validate = ajv.compile(vectorsSchema)
 function validateTestVectors() {
   const entries = fs.readdirSync(vectorsDir, { withFileTypes: true })
 
+  function validateDescriptions(testData) {
+    // accumulate duplicates inside a map
+    const descriptions = new Set()
+    for (const vector of testData.vectors) {
+      if (descriptions.has(vector.description)) {
```

Comment:
is this the approach taken across all vectors - uniqueness is by description?

Reply:
@decentralgabe I think what you're getting at is that it may not be intuitive that `description` must be unique since in many contexts descriptions are free form text fields that don't have a uniqueness constraint, correct?

Since only a few test vectors have been committed thus far, would it make sense to either:
- add an `id` property that is unique so that the `description` doesn't have a uniqueness constraint
- rename the `description` property to `id` or `testId`?

Reply:
what's the benefit of having a unique identifier per vector anyway?

Reply:
I'm on the fence if this matters, but if I had to throw out potential benefits...
- makes it easier to figure out which test failed if they have unique names
-  if there are test frameworks that don't allow test identifiers to be duplicated?

Reply:
identifying a vector is useful for...
1. knowing you implement the same vector
2. reporting

Reply:
Another benefit is that having a unique identifier according to the behavior that is being tested acts as a self documentation. It's a typical best practice of unit tests. Take a look at the `description` values that I've added for the `did:web` example.